src/mam4xx/convproc.hpp

```diff
@@ -92,6 +92,8 @@ class ConvProc {
   };
   // maxd_aspectype = maximum allowable number of chemical species
   // in each aerosol mode.
+  //
+  // TODO: Why is this not 7?
```

Comment:
I'm guessing this accounts for interstitial and cloudborne species...?

Reply:
I'm not sure.  So far for all of the arrays that are dimensioned by this value of 14, only 7 have been used. But I am not finished with all of the functions, maybe the use of 14 instead of 7 sill become apparent.